README.md

```diff
@@ -303,4 +303,41 @@ Make sure the subscription id, resource group, storage and container strings are
 
 **Do Not** commit your .terraform/terraform.tfstate or .tfvars file to version control. 
 
+## Logging
+
+### Local logs
+
+Local logging is written to a a ToDo_Flask.log file.  It currently logs at the INFO level.  Add this file to .gitignore, do not commit to source control. 
+
+
+### Log Aggregation
+
+Log aggregation is provided by [Loggly](https://loggly.com).  You'll need to sign up for a Loggly account, and retrieve your customer token from Loggly. Setting up logging for Python apps in covered [here](https://documentation.solarwinds.com/en/success_center/loggly/content/admin/python-http.htm?cshid=loggly_python-http).
+
+You'll need a ```python.conf``` file in the root of the project:
+
+```python
+[handlers]
```

Comment:
This approach is quite neat. You will need to add a way to configure at least the log level via environment variable though, as per submission checklist **The log level is configurable via an environment variable**

Reply:
Added log level override via environment variable.